packages/sdks/snippets/angular-16/src/app/home/homepage.component.ts

```diff
@@ -0,0 +1,32 @@
+import { CommonModule } from '@angular/common';
+import { Component } from '@angular/core';
+import {
+  Content,
+  fetchOneEntry,
+  type BuilderContent,
+} from '@builder.io/sdk-angular';
+
+@Component({
+  selector: 'app-homepage',
+  standalone: true,
+  imports: [Content, CommonModule],
+  template: `
+    <ng-container *ngIf="!content">404</ng-container>
+    <builder-content
+      *ngIf="content"
+      [model]="'homepage'"
+      [content]="content"
+      [apiKey]="'ee9f13b4981e489a9a1209887695ef2b'"
+    ></builder-content>
```

Comment:
instead of writing it like this, i think we should write it as an if else statement similar to others (not use two ngIfs)

for this you can do: `*ngIf="content; else elseBlock"` and an ng-template with 404. Check: https://angular.dev/api/common/NgIf

Reply:
and also you're missing the isPreviewing case in Angular

`!content && !isPreviewing()`